packages/frontend/src/hooks/useRealityAnswer.ts

```diff
@@ -0,0 +1,52 @@
+import { BigNumber, Contract } from "ethers";
+import { useEffect, useState } from "react";
+import { useNetwork, useProvider } from "wagmi";
+import { REALITY_CONTRACT_BY_CHAIN, SupportedChain } from "../commons";
+import REALITY_ETH_V3_ABI from "../abis/reality-eth-v3.json";
+
+export function useRealityAnswer(questionId: string | undefined): {
+    loading: boolean;
+    data: BigNumber | undefined;
+} {
+    const { chain } = useNetwork();
+    const provider = useProvider();
+
+    const [loading, setLoading] = useState(true);
+    const [data, setData] = useState<BigNumber>();
+
+    useEffect(() => {
```

Comment:
This hook could probably be rewritten as a wrapper around wagmi's `useContractRead` hook